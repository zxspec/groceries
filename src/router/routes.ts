import type { Route } from "router5";

const routes: Route[] = [
  { name: "home", path: "/" },
  { name: "details", path: "/details/:product" },
  { name: "404", path: "/404" },
];

export default routes;
