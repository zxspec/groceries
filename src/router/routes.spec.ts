import routes from "./routes";

describe("routes", () => {
  it("should have correct routes", () => {
    expect(routes).toMatchSnapshot();
  });
});
