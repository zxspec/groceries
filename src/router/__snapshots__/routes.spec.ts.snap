// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes should have correct routes 1`] = `
Array [
  Object {
    "name": "home",
    "path": "/",
  },
  Object {
    "name": "details",
    "path": "/details/:product",
  },
  Object {
    "name": "404",
    "path": "/404",
  },
]
`;
